wp-content/themes/core/components/contentblock/contentblock.twig

```diff
@@ -28,9 +28,9 @@ Attributes:
     }
  #}
 
-<div {{ classes }}>
+<{{ tag }} {{ classes }}>
```

Comment:
whats content block vs card these days?

Reply:
Per our convo yesterday, this will go away and become a card variant, I'm planning to tackle that as part of my hero work 🤘